js/controller.js

```diff
@@ -578,8 +574,10 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
     },
 
     onBitrateInfoAvailable: function(event, bitrates) {
-      this.state.videoQualityOptions.availableBitrates = bitrates;
-      this.renderSkin({"videoQualityOptions": {"availableBitrates": bitrates}});
+      if (bitrates && bitrates.bitrates) {
+        this.state.videoQualityOptions.availableBitrates = bitrates.bitrates;
+        this.renderSkin({"videoQualityOptions": {"availableBitrates": bitrates.bitrates }});
+      }
```

Comment:
Yes, and bitrates.bitrates is an array of objects that satisfy this format.  We just had to wrap them into another object so that message bus won't split this array into many arguments.  So, Xeniq correctly mentioned that flash plugin should also pass to message bus such array wrapped in the object of the same format